### PR TITLE
Ws 119 separar paquetes

### PR DESCRIPTION
**Pull request by fedeaviles**
**Type of MR:**

- [x] Feature
- [ ] Fix
- [ ] HotFix

**Small description:**
Converted to monorepo.
Packages:
- sdk
- sdk-nft
- sdk-storage-aws
- sdk-storage-http
- sdk-storage-ipfs

# Checklist:

- [x] My code follows the styleguides of the project (successfully pass linters)
- [x] I tested my code.
- [x] I commented/docummented my code in a way that complex parts are explained.
- [ ] If required, i updated project's README file.
- [ ] I added/updated tests that tests my changes in code.
- [x] All previous and new tests pass.
- [ ] I commit any new dependency.
- [ ] I updated UML diagram in confluence if any model was modified in this MR.

### CODE REVIEW

File: packages/nft/package.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    "typesVersions": {
+      "*": {
+        ".": [
+          "dist/types/index.d.ts"
+        ],
+        "types": [
+          "dist/types/types/index.d.ts"
+        ]
+      }
+    },
```
Comment (mauricio-b-r) [inline packages/nft/package.json:39]: missing storage?

File: packages/nft/src/NFTFactory.ts [hunk 2]
```diff
@@ -1,11 +1,13 @@
 /* eslint-disable @typescript-eslint/no-non-null-assertion */
-import { isRequired } from '../utils/validations'
 import StorageType from './storage/StorageType'
 import { NFT_METADATA_FORMAT } from './types'
 import { NFT } from './NFT'
-import { NFT721Manager } from '../managers/NFT721Manager'
-import errors from '../types/errors'
-import { NftMetadata } from '../types/interfaces'
+import { NFT721Manager } from '@ripio/sdk'
+import { errors } from '@ripio/sdk'
```
Comment (mauricio-b-r) [inline packages/nft/src/NFTFactory.ts:6]: merge into one import statement

File: packages/nft/src/__tests__/NFTFactory.test.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-import Resource from '../../nft/storage/Resource'
-import StorageType from '../../nft/storage/StorageType'
-import errors from '../../types/errors'
-import { NFT721Manager } from '../../managers/NFT721Manager'
+import { NftMetadata } from '../types/interfaces'
+import { NFT, NFTFactory, NFT_METADATA_FORMAT } from '../'
+import Resource from '../storage/Resource'
+import StorageType from '../storage/StorageType'
+import { errors } from '@ripio/sdk'
+import { NFT721Manager } from '@ripio/sdk'
```
Comment (mauricio-b-r) [inline packages/nft/src/__tests__/NFTFactory.test.ts:6]: merge into one import statement

File: packages/sdk/package.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
+{
+    "name": "@ripio/sdk",
+    "version": "1.1.0",
```
Comment (mauricio-b-r) [inline packages/sdk/package.json:3]: the sdk package should increase to version 2.0, right?

File: packages/sdk/package.json [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+        "require": "./dist/cjs/types/index.js",
+        "import": "./dist/esm/types/index.js",
+        "default": "./dist/esm/types/index.js"
+      },
+      "./gaming": {
+        "types": "./dist/types/gaming/index.d.ts",
+        "require": "./dist/cjs/gaming/index.js",
+        "import": "./dist/esm/gaming/index.js",
+        "default": "./dist/esm/gaming/index.js"
+      },
```
Comment (mauricio-b-r) [inline packages/sdk/package.json:46]: I think that the gaming one is not used anymore

File: packages/sdk/package.json [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+        ],
+        "utils": [
+          "dist/types/utils/index.d.ts"
+        ],
+        "types": [
+          "dist/types/types/index.d.ts"
+        ],
+        "gaming": [
+          "dist/types/gaming/index.d.ts"
+        ],
```
Comment (mauricio-b-r) [inline packages/sdk/package.json:79]: missing storage, and should the gaming one be deleted?

File: packages/sdk/src/index.ts [hunk 7]
```diff
@@ -3,4 +3,3 @@ export * from './gaming'
 export * from './utils'
 export * from './types'
 export * from './connectors'
-export * from './nft'
```
Comment (mauricio-b-r) [inline packages/sdk/src/index.ts:6]: should the gaming one be also removed? (on top)

File: package.json [hunk 8]
```diff
@@ -1,176 +1,49 @@
...
-        "dist/types/gaming/index.d.ts"
-      ],
-      "ethers-utils": [
-        "dist/types/utils/ethersUtils.d.ts"
-      ]
-    }
-  },
+  "name": "ripio",
+  "version": "0.0.0",
+  "license": "MIT",
```
Comment (mauricio-b-r) [inline package.json:4]: Is there a reason for having the base package as MIT and the others as Apache 2.0?

Comment (fedeaviles) [reply package.json]: No, updated

File: packages/nft/package.json (hunk 1 above)
Comment (fedeaviles) [reply packages/nft/package.json]: added

File: packages/nft/src/NFTFactory.ts (hunk 2 above)
Comment (fedeaviles) [reply packages/nft/src/NFTFactory.ts]: fixed

File: packages/nft/src/__tests__/NFTFactory.test.ts (hunk 3 above)
Comment (fedeaviles) [reply packages/nft/src/__tests__/NFTFactory.test.ts]: fixed

File: packages/sdk/package.json (hunk 4 above)
Comment (fedeaviles) [reply packages/sdk/package.json:3]: Yes, I'm trying to figure out if it can be done with semantic release

File: packages/sdk/package.json (hunk 5 above)
Comment (fedeaviles) [reply packages/sdk/package.json:46]: Yes, but that pr is not merged. Maybe I should remove it in this one

File: packages/sdk/package.json (hunk 6 above)
Comment (fedeaviles) [reply packages/sdk/package.json:79]: Storage is not part of the main package (sdk) anymore. It is on the nft package.

File: packages/sdk/src/index.ts (hunk 7 above)
Comment (fedeaviles) [reply packages/sdk/src/index.ts:6]: Answered above

**Pull request merged**